homeassistant/components/ombi/__init__.py

```diff
@@ -0,0 +1,149 @@
+"""Support for Ombi."""
+import logging
+
+import pyombi
+import voluptuous as vol
+
+from homeassistant.const import (
+    CONF_API_KEY,
+    CONF_HOST,
+    CONF_PORT,
+    CONF_SSL,
+    CONF_USERNAME,
+)
+import homeassistant.helpers.config_validation as cv
+
+from .const import (
+    CONF_URLBASE,
+    CONF_NAME,
+    CONF_SEASON,
+    DEFAULT_PORT,
+    DEFAULT_SEASON,
+    DEFAULT_SSL,
+    DEFAULT_URLBASE,
+    DOMAIN,
+    SERVICE_MOVIE_REQUEST,
+    SERVICE_MUSIC_REQUEST,
+    SERVICE_TV_REQUEST,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def urlbase(value) -> str:
+    """Validate and transform urlbase."""
+    if value is None:
+        raise vol.Invalid("string value is None")
+    value = str(value).strip("/")
+    if not value:
+        return value
+    return value + "/"
+
+
+SUBMIT_MOVIE_REQUEST_SERVICE_SCHEME = vol.Schema({vol.Required(CONF_NAME): cv.string})
+
+SUBMIT_MUSIC_REQUEST_SERVICE_SCHEME = vol.Schema({vol.Required(CONF_NAME): cv.string})
+
+SUBMIT_TV_REQUEST_SERVICE_SCHEME = vol.Schema(
+    {
+        vol.Required(CONF_NAME): cv.string,
```

Comment:
We only use `CONF_*` constants for config keys. For service data keys we use `ATTR_*` constants.